template.yaml

```diff
@@ -106,8 +106,8 @@ Resources:
                 httpMethod: POST
                 type: AWS_PROXY
               responses:
-                '200':
-                  description: OK
+                '202':
+                  description: ACCEPTED
```

Comment:
We can use a more human readable description here, for example "Publication modified successfully".